next.config.js

```diff
@@ -1,15 +1,15 @@
 const { withContentlayer } = require("next-contentlayer");
-const redirects = require("./redirects"); 
+const redirects = require("./redirects");
```

Comment:
Not needed for this pr, but I have been convinced that alias paths are the way forward.

Reply:
Yeah they're used in some places but not everywhere.